cluster/juju/layers/kubernetes-worker/templates/kube-proxy-kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+clusters:
+  - cluster:
+      certificate-authority: /etc/kubernetes/ca.pem
```

Comment:
ca.crt is the file name generated by easyrsa. I can change it to ca.pem if needed, but we should decide on one name.